libafl/src/inputs/mod.rs

```diff
@@ -166,6 +166,39 @@ pub trait HasMutatorBytes: HasLen {
     }
 }
 
+impl HasMutatorBytes for &mut Vec<u8> {
+    fn bytes(&self) -> &[u8] {
+        self
+    }
+
+    fn bytes_mut(&mut self) -> &mut [u8] {
+        self
+    }
+
+    fn resize(&mut self, new_len: usize, value: u8) {
+        Vec::resize(self, new_len, value);
+    }
+
+    fn extend<'a, I: IntoIterator<Item = &'a u8>>(&mut self, iter: I) {
+        Vec::extend(self, iter);
+    }
+
+    fn splice<R, I>(&mut self, range: R, replace_with: I) -> Splice<'_, I::IntoIter>
```

Comment:
@addisoncrump I guess this gets really hairy because you cannot use vecs normally anymore if the trait is in scope, right? Should I instead create a wrapper type and `From` here?

Reply:
Removed it / replaced it with a wrapper type.